src/test/shared/telemetry/defaultTelemetryService.test.ts

```diff
@@ -86,24 +92,23 @@ describe('DefaultTelemetryService', function () {
         assert.strictEqual(mockPublisher.feedback, feedback)
     })
 
-    it('assertPassiveTelemetry() throws if active, non-cached metric is emitted during startup', async function () {
-        // Simulate cached telemetry by prepopulating records before start().
-        // (Normally readEventsFromCache() does this.)
```

Comment:
I do not understand why these comments were removed.

Reply:
Because the cache is no longer being taken into account in the function.